README.md

```diff
@@ -4,12 +4,16 @@ Generates a basic analysis of a CSV to an XLSX workbook.
 ```
 usage: csv_summary [-h] [-o OUTPUT_PATH]
                    [--category-threshold CATEGORY_THRESHOLD]
-                   [--date-format DATE_FORMAT] [-i IGNORE_VALUE]
-                   [-s NUM_SAMPLES]
-                   csv_path
+                   [--date-format DATE_FORMAT]
+                   [--date-time-format DATE_TIME_FORMAT] [-i IGNORE_VALUE]
```

Comment:
Can you include more than one ignore value, e.g. 'na', '?', 'n/a' ? If not, a reasonable way to do this is to have them separated by commas.

Reply:
Yes, you can include any number by supplying the option more than once, e.g. `-i '?' -i 'N/A'`. I will mention this explicitly.